pkg/controller/infinispan/resources/operatorconfig/rice-box.go

```diff
@@ -0,0 +1,44 @@
ndor_cache_manager_default_number_of_cache_configurations,namespace)\",\n        \"refresh\": 1,\n        \"regex\": \"\",\n        \"skipUrlSync\": false,\n        \"sort\": 0,\n        \"tagValuesQuery\": \"\",\n        \"tags\": [],\n        \"tagsQuery\": \"\",\n        \"type\": \"query\",\n        \"useTags\": false\n      },\n      {\n        \"allValue\": null,\n        \"current\": {\n          \"selected\": true,\n          \"tags\": [],\n          \"text\": \"All\",\n          \"value\": [\n            \"$__all\"\n          ]\n        },\n        \"datasource\": \"Prometheus\",\n        \"definition\": \"label_values(vendor_cache_manager_default_number_of_cache_configurations{namespace=~\\\"$namespace\\\"}, service)\",\n        \"hide\": 0,\n        \"includeAll\": true,\n        \"label\": \"Cluster\",\n        \"multi\": true,\n        \"name\": \"cluster\",\n        \"options\": [],\n        \"query\": \"label_values(vendor_cache_manager_default_number_of_cache_configurations{namespace=~\\\"$namespace\\\"}, service)\",\n        \"refresh\": 1,\n        \"regex\": \"\",\n        \"skipUrlSync\": false,\n        \"sort\": 0,\n        \"tagValuesQuery\": \"\",\n        \"tags\": [],\n        \"tagsQuery\": \"\",\n        \"type\": \"query\",\n        \"useTags\": false\n      },\n      {\n        \"allValue\": null,\n        \"current\": {\n          \"selected\": true,\n          \"tags\": [],\n          \"text\": \"All\",\n          \"value\": [\n            \"$__all\"\n          ]\n        },\n        \"datasource\": \"Prometheus\",\n        \"definition\": \"query_result({job=~\\\"$cluster\\\", namespace=~\\\"$namespace\\\", __name__=~\\\"vendor_cache_manager_default_cache_.*_cluster_cache_stats_activations\\\"})\",\n        \"hide\": 0,\n        \"includeAll\": true,\n        \"label\": null,\n        \"multi\": true,\n        \"name\": \"caches\",\n        \"options\": [],\n        \"query\": \"query_result({job=~\\\"$cluster\\\", namespace=~\\\"$namespace\\\", __name__=~\\\"vendor_cache_manager_default_cache_.*_cluster_cache_stats_activations\\\"})\",\n        \"refresh\": 1,\n        \"regex\": \"/vendor_cache_manager_default_cache_(.*)_cluster_cache_stats_activations/\",\n        \"skipUrlSync\": false,\n        \"sort\": 0,\n        \"tagValuesQuery\": \"\",\n        \"tags\": [],\n        \"tagsQuery\": \"\",\n        \"type\": \"query\",\n        \"useTags\": false\n      }\n    ]\n  },\n  \"time\": {\n    \"from\": \"now-30m\",\n    \"to\": \"now\"\n  },\n  \"timepicker\": {\n    \"refresh_intervals\": [\n      \"10s\",\n      \"30s\",\n      \"1m\",\n      \"5m\",\n      \"15m\",\n      \"30m\",\n      \"1h\",\n      \"2h\",\n      \"1d\"\n    ],\n    \"time_options\": [\n      \"5m\",\n      \"15m\",\n      \"1h\",\n      \"6h\",\n      \"12h\",\n      \"24h\",\n      \"2d\",\n      \"7d\",\n      \"30d\"\n    ]\n  },\n  \"timezone\": \"\",\n  \"title\": \"Infinispan\",\n  \"uid\": \"U826WzBGz\",\n  \"version\": 35\n}"),
```

Comment:
Having whole dashboard json file in single line doesn't look practical. Also doesn't it just duplicate `grafana_dashboard.json` file?

Reply:
This is generator called rice. File should be identical to original JSON. I will add generator calling script later and validate against env. Storing JSON as const as right now looks like a long string, not JSON formatted

Reply:
@Crumby I've validated on OCP env and Grafana Dashboard JSON seems to be looks like better, than JSON as string const
![image](https://user-images.githubusercontent.com/24249373/114179095-3cfeee80-9947-11eb-8b3b-b746d77af8c0.png)